VladimirPopov_WebForms/templates/webforms/form/default.phtml

```diff
@@ -21,11 +21,11 @@ if (!empty($tmpData['field']) && is_array($tmpData['field'])) {
     }
 }
 if ($this->getForm()->getIsActive() == \VladimirPopov\WebForms\Model\Form::STATUS_DISABLED || !$this->isDirectAvailable()): ?>
-    <?= $this->getNotAvailableMessage(); ?>
+    <?= $block->escapeHtml($this->getNotAvailableMessage()); ?>
```

Comment:
are we sure here won't be posted HTMl message just raw text?

Reply:
No, actually can be. Changed.